spec/models/user_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Happen', photo: 'google.com', bio: 'I was born in 1900', posts_counter: '5') }
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts counter should be greater than 0' do
+    subject.posts_counter = -5
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts counter should be equal to 0' do
+    subject.posts_counter = 0
+    expect(subject).to be_valid
+  end
+
+  it 'should list three recent posts' do
+    4.times do |i|
+      Post.create(title: 'Hello', text: "This is my #{i}th post", comments_counter: 0, likes_counter: 0,
+                  author: subject)
+    end
+
+    expect(subject.three_recent_posts[0].text).to eq 'This is my 3th post'
+    expect(subject.three_recent_posts.length).to eq 3
+  end
+end
```

Comment:
- [ ] Please fix your tests that are failing
![s](https://user-images.githubusercontent.com/44745855/188285539-b33cf93f-461e-4ba9-b62d-b7edc493f2eb.png)

Reply:
Hi @DeliceLydia, Iran the test and they all succeed 
here is a screenshot of my test.
![image](https://user-images.githubusercontent.com/56921581/188286265-8cf25c9a-134e-4fc5-b0c3-1524f30c3a8a.png)